spec/datadog/profiling/exporter_spec.rb

```diff
@@ -18,6 +19,7 @@
   let(:code_provenance_data) { 'dummy code provenance data' }
   let(:pprof_recorder_serialize) { [start, finish, pprof_data] }
   let(:pprof_recorder) { instance_double(Datadog::Profiling::StackRecorder, serialize: pprof_recorder_serialize) }
+  let(:worker) { instance_double('Datadog::Profiling::Collectors::CpuAndWallTimeWorker', stats_and_reset: stats) }
```

Comment:
Ahhhhhh I thought the reference was because the native extension may not be loaded, I forgot that we had a couple where we were using extremely modern (cof) syntax and can't load the class at all.

Thanks for clarifying it -- I should've tested myself :sweat_smile: before suggesting lol.

Reply:
Actually -- may be worth leaving here a `TODO` saying we can clean this up when we drop support for old Rubies :)